### PR TITLE
[AND-3104] Vungle 6.8.1

### PR DESCRIPTION
**Pull request by jiancchen**
Updating Android to 6.8.1.
No major changes in adapter.
Added no op new onAdViewed Callback

### CODE REVIEW

**Review by HuFangshuai — APPROVED** [review]
LGTM

**Comment by HuFangshuai** [comment]
Should we change the target branch?

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java [hunk 1]
```diff
@@ -335,6 +335,11 @@ public void run() {
         });
   }
 
+  @Override
+  public void onAdViewed(String id) {
+    // no op for now
```
Comment (palrahul) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java:340]: can we change the wording as "no-op , to be mapped to respective adapter events in future release"

**Review by palrahul — APPROVED** [review]
lgtm

**Comment by alexander-gubov** [comment]
Isn't master is an original repos's master?

**Pull request closed without merging**